cassdegrees/ui/views.py

```diff
@@ -95,3 +95,66 @@ def sampleform(request):
 
 def create_subplan(request):
     return render(request, 'createsubplan.html')
+
+
+# inspired by the samepleform function creatd by Daniel Jang
+def manage_courses(request):
+    courses = requests.get(request.build_absolute_uri('/api/model/course/?format=json')).json()
```

Comment:
```suggestion
    courses = requests.get(request.build_absolute_uri('/api/model/course/?format=json')).json()
    courses = [{'code': course} for course in  set([x['code'] for x in courses])]
```

Convert courses to a unique set of only course codes. Please note that this is NOT a good solution and is intended to be a single line fix. Best way might be to try do a SQL Query for unique course codes, consult @Daniel-Jang for this

Reply:
will do